docs/hugo/content/guide/metrics.md

```diff
@@ -7,7 +7,7 @@ The metrics exposed fall into two groups: Azure based metrics, and reconciler me
 
 ## Toggling the metrics
 
-By default, metrics for ASOv2 are turned on and can be toggled by the following options:
+By default, secure metrics for ASOv2 are turned on and can be toggled by the following options:
```

Comment:
Questions based on our chat:
1. Do we actually need to expose an `8080` service - it seems that CAPI doesn't? I think, you should be able to scrape the pods metrics from within the cluster without a service. Maybe we need to do a little bit more reading/learning on if we really should have a service exposed for metrics or not.
2. Is the right default, to have securemetrics= false - I sorta wonder if we should set the default to true. I acknowledge that this is _not_ what we were doing before, but it is I think what we _thought_ we were doing before. I think etiher way is breaking because before we were exposing 2 services, an 8443 AND an 8080.

Reply:
Just maybe doublecheck other projects doing this... which ones have services and which ones don't? Or we can talk with John Rusk or somebody on his team, they may have suggestions here.